python/src/tasks/doublet_score.py

```diff
@@ -1,14 +1,15 @@
 import json
 from config import get_config
 from result import Result
+import numpy as np
 import requests
 
 config = get_config()
 
 
 class GetDoubletScore:
-    def __init__(self, msg):
-        self.task_def = msg["body"]
+    def __init__(self):
+        self.task_def = "Doublet-score"
```

Comment:
As the task does not take any more parameter from `msg`, I think you can just delete this line.